uf3/data/composition.py

```diff
@@ -186,12 +188,12 @@ def sort_interaction_map(imap: Dict[Tuple[str], Any]) -> Dict[Tuple[str], Any]:
     return {sort_interaction_symbols(k): v for k, v in imap.items()}
 
 
-def sort_interaction_symbols(symbols: Collection[str]) -> Tuple:
+def sort_interaction_symbols(symbols: Collection[str], fix_first=True) -> Tuple:
```

Comment:
Please add doc string for fix_first

Reply:
@JasonGibsonUfl Could you please add it? This has already been merged to `uf3:element_ordering`, and I have no write access to this branch anymore. Here is the doc string:

    Args:
        symbols (tuple): tuple or list of element symbols
        fix_first (bool): whether to fix the first element in symbols
            (default: True)
            True should be used for many-body interactions (i.e. >2)
            False should be used for sorting the element_list

    Returns:
        tuple of sorted element symbols